app/models/miq_policy_content.rb

```diff
@@ -3,6 +3,53 @@ class MiqPolicyContent < ApplicationRecord
   belongs_to :miq_event_definition
   belongs_to :miq_action
 
+  def self.seed
+    fixture_file = File.join(FIXTURE_DIR, "miq_policy_contents.yml")
+    policy_contents = File.exist?(fixture_file) ? YAML.load_file(fixture_file) : []
+
+    policy_contents.each do |content|
+      find_associations!(content)
+      policy_name = content[:miq_policy].try(:name)
+
+      rec = find_by_pkey(policy_name, content[:success_sequence], content[:failure_sequence])
+      if rec.nil?
+        create_seed_record(policy_name, content)
+      else
+        update_seed_record(rec, policy_name, content)
```

Comment:
I probably wouldn't create methods for the create/update work.  If I were to extract a method, I would create a method to seed one content...

```ruby
...
policy_contents.each do |content|
  seed_one_policy_content(content)
end
```